index.d.ts

```diff
@@ -122,9 +122,10 @@ declare namespace WAWebJS {
          * Request authentication via pairing code instead of QR code
          * @param phoneNumber - Phone number in international, symbol-free format (e.g. 12025550108 for US, 551155501234 for Brazil)
          * @param showNotification - Show notification to pair on phone number
+         * @param intervalMs - The interval in milliseconds on how frequent to generate pairing code (WhatsApp default to 3 minutes)
          * @returns {Promise<string>} - Returns a pairing code in format "ABCDEFGH"
          */
-        requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>
+        requestPairingCode(phoneNumber: string, showNotification = true, intervalMs = 180000): Promise<string>
```

Comment:
`requestPairingCode` method shoud be correcly declared.
Because abstract functions cannot have function default parameters.

✅ Correct way:
`requestPairingCode(phoneNumber: string, showNotification: boolean, intervalMs: number): Promise<string>`

❌ Incorrect way:
`requestPairingCode(phoneNumber: string, showNotification = true, intervalMs = 180000): Promise<string>`